js/pages/characterizations/components/characterizations/characterization-view-edit/characterization-design.js

```diff
@@ -132,6 +133,15 @@ define([
                 return `<a href='#' data-bind="click: () => $component.params.${action}('${d[identifierField]}')">Remove</a>`;
             }
         }
+        
+        getClassName(isNameEmpty){
+            this.disableSave(isNameEmpty);
```

Comment:
`getClassName` which changes observable??

add computed based on design and updated passed variable via subscription